optimize/backend/src/it/java/io/camunda/optimize/test/it/extension/ZeebeExtension.java

```diff
@@ -283,7 +283,9 @@ private void setZeebeRecordPrefixForTest() {
   }
 
   private void destroyClient() {
-    zeebeClient.close();
-    zeebeClient = null;
+    if (zeebeClient != null) {
+      zeebeClient.close();
+      zeebeClient = null;
+    }
```

Comment:
I noticed that we were trying to close the client, even in the case when tests weren't set up correctly in the first place. While it wasn't really hurting us, this is just a simple check to avoid the NPE case